### ISSUE TITLE
Calendar - Voiceover users are unable to move to the dates in the calendar view

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#19468]
### Environment Information

- **Package version(s)**: latest
- **Browser and OS versions**: Chrome on iOS 14.7.1

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar

#### Actual behavior:

While oviceover is on for iOS users, date cells in the calendar control are not reachable when navigate by swiping. So users miss out on the information that is available for other users and they are not able to select the date.

#### Expected behavior:

users should be able to navigate to each date cell.

**Comment by gouttierre** [comment]
@yanbxice - Thanks for filing this issue with us. To set expectation our team will have a look at the issue but only once capacity allows.
  
@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by sbarrameda** [comment]
@smhigley Hi Sarah,

Wanted to bring this to your attention again. We are able to repro this issue in version 7.19 but not in the latest Fabric version.

**Comment by JustSlone** [comment]
> @smhigley Hi Sarah,
> 
> Wanted to bring this to your attention again. We are able to repro this issue in the sharing dialog of the Teams app which is using version 7.19 but not in the latest Fabric version.

I would recommend updating to the latest version of Fabric, if you know specifically the fix that resolved this issue we could consider backporting it, but that is expensive. Folks need to stay updated to the latest versions in order to continue getting fixes.
